src/frontend/src/pages/Discover.tsx

```diff
@@ -1,4 +1,25 @@
+import { useEffect } from "react";
+import { useAppDispatch, useAppSelector, type RootState } from "../store/store";
+import { getNeighborhoodsAndBounds } from "src/features/neighborhoods/neighborhoodsThunk";
```

Comment:
I'd consider adding store and features to [here](https://github.com/azavea/echo-locator/blob/develop/src/frontend/vite.config.ts#L18) and [here](https://github.com/azavea/echo-locator/blob/develop/src/frontend/tsconfig.app.json#L28) to have simpler imports and avoid relative imports. If we go with this solution, I'd update the relative imports in other modules/files.

Additionally, I am not familiar with the naming convention of using `features` under `src`. Could you expand on it a bit?